ophyd/tests/test_epicspath.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from ..areadetector.paths import EpicsPathSignal
+
+
+def test_path_semantics_exception():
```

Comment:
It's probably not worth making a new test faile just for this one test. Could this be moved to `test_signal.py`?